### DISCUSSION TITLE
How to archive specific chats (or just hide them)?

### DISCUSSION THREAD

**Discussion by Aidavdw** [d99kris/nchat#310] [Q&A]
To everyone, there is always that one chat that keeps spamming. It might be nice to be able to separate those chats out to reduce the information clutter. [Whatsmeow should already be able to raise an event like this](https://github.com/d99kris/nchat/blob/dbfcc3d1428cb4ad3352e50cabc481233569e34f/lib/wmchat/go/ext/whatsmeow/types/events/appstate.go#L85), but a more general 'hide chat' would be just as effective.

**Comment by d99kris** [comment]
Hi @Aidavdw - would muting the chat (in the official WhatsApp app) work for you? And then in nchat using options like
```
muted_indicate_unread=0
muted_position_by_timestamp=0
```
to avoid seeing messages from the "one chat that keeps spamming"?
